### DISCUSSION TITLE
Iv'e just received a weird email making me owner of an organization I don't recognize.

### DISCUSSION THREAD

**Discussion by shawty** [community/community#120769] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Don't really know where to put this, but I've just received the following email:

![image](https://github.com/community/community/assets/1876454/5e01c9f4-e8b3-4299-8440-c174c6c4513f)

The email domain originating things, seems to be legit:

![image](https://github.com/community/community/assets/1876454/8cde2d7d-5178-4f6a-a4b2-31192f58445a)

and the email address this has been sent to, is my registered email address for my own github repos page at github.com/shawty

When I look at the organisation, I don't recognise the "billing email", and there are no repos, or other people in the organisation, other than myself.

I most certainly have not created this organisation, and I'm not aware of anyone creating it for me, or for any of the projects I'm currently working on.

I do however appear to have full rights to perform any action I want on this organization.

Anyone got any ideas what's going on here?

GitHub bug?
Mistyped email address?
Attempted Phishing Exercise?

Cheers
Shawty

**Comment by MindOfBear** [comment]
Hello @shawty,

By the links in the screenshot, for me, its not looking like phishing. If you can acces the organization from you github profile (without any click on the email), i recommend you to notify Github in order to check the case. If you can't see the organization in your github profile, i recommend you to just ignore the email.

Cheers,
Ovidiu

**Reply by shawty** [reply]
Hi @MindOfBear,

I just checked my "organisations" from my profile menu, and yup, the organisation does indeed show up, with me as the owner, and without involving the email.

Thing is, I was actually trying to notify GitHub when I ended up here.  They kind of make it quite hard to tell them anything unless your a paying customer or on one of their support plans.

Any pointers you have on how to notify them of this, would be greatly appreciated.

Cheers
Shawty

**Reply by ebndev** [reply]
Hi @shawty,

Since you have access to the organization as an owner, I would recommend deleting the organization or you also have the option to leave it.

While this community is managed by GitHub employees, it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
